contrib/hyperkube/Makefile

```diff
@@ -3,6 +3,8 @@ IMAGE?=sapcc/hyperkube
 ARCH=amd64
 CNI_RELEASE=v0.6.0
 CNI_PLUGINS_RELEASE=v0.7.1
+CNI_DEDUP_BRIDGE_RELEASE=v0.1.0
+MULTUS_RELEASE=v3.1
```

Comment:
no 3.2?

Reply:
yes i want to replicate what we have installed prior to upgrading to unknown